freqtrade/exchange/exchange.py

```diff
@@ -216,8 +216,9 @@ def get_quote_currencies(self) -> List[str]:
         Return a list of supported quote currencies
         """
         markets = self.markets
-        currencies = set([x.get('quote') for _, x in markets.items()])
-        return list(currencies)
+        currencies = list(set([x.get('quote') for _, x in markets.items()]))
+        currencies.sort()
+        return currencies
```

Comment:
* Proposing small improvement:
`return sorted(set([x.get('quote') for _, x in self.markets.items()]))`

* Do you really think .get() is needed here? If so, this should probably (not in this PR) be fixed in get_markets() as well...